### DISCUSSION TITLE
Pose label?

### DISCUSSION THREAD

**Discussion by jurieltoribio** [talmolab/sleap#752] [Help!] [answered]
Good day! 
So I'm creating a detection behavior for rabbits. and I'd like to capture this specific pose of them that they are resting.
How can I label that on that specific pose of the skeleton of a rabbit resting? I'm planning to use also this for detection in real-time (get frames -> predict frames -> output 'resting').

![image](https://user-images.githubusercontent.com/22687386/169280502-bb7d721e-8568-4561-80df-99bbbf27b12e.png)

**Comment by roomrys** [comment, marked as answer]
Hi @jurieltoribio,

Unfortunately, behavioral segmentation is not currently supported in SLEAP. However, some options to consider are either annotating behaviors in BENTO or using SimBA for behavioral classification. Both BENTO and SimBA support SLEAP tracking data.

BENTO requires manual annotations whereas SimBA uses machine learning for behavioral segmentation. SimBA was intended to classify social behavior (2+ animals), so I am not sure how well it will work if you are only working with a single animal - it might be worth a shot! You can also use BENTO with scikit-learn to fit a basic classifier to the SLEAP tracking data (check out this [notebook on how to work with SLEAP analysis files](https://colab.research.google.com/github/talmolab/sleap/blob/main/docs/notebooks/Analysis_examples.ipynb)).

Hope this helps point you in the right direction.

Thanks,
Liezl

**Comment by sronilsson** [comment]
SimBA or BENTO should work, don’t know much about rabbits but there are freezing classifiers about e.g, 
https://www.biorxiv.org/content/10.1101/2022.03.28.486055v1.full.pdf
Simple behaviors like freezing are often be solved by heuristics e.g. body-parts not moving == animal freezing, and no additional packages like necessary. But again don't know much about rabbit freezing, perhaps its more complicated 😬
